text-to-speech/versta/export/__main__.py

```diff
@@ -86,7 +96,26 @@ def main(
     quantization_dir.mkdir(parents=True, exist_ok=True)
 
     # Step 1: Convert the model to ONNX format
-    convert_model_to_onnx(model, converted_dir, model_format)
+    convert_model_to_onnx(model, converted_dir, model_format, sub_voice)
+
+    # For Piper models, update output directory based on language information from config
+    if model_format == "piper":
+        config_file = converted_dir / "config.json"
+        if config_file.exists():
+            from .convert_piper_to_onnx import get_language_info_from_config
+            language_info = get_language_info_from_config(config_file)
+            if language_info and "name_english" in language_info:
+                # Create a language-specific output directory if not already specified
+                # Use the original output_dir as parent, and add language-specific folder
+                language_name = language_info["name_english"].lower().replace(" ", "_")
+                
+                # Only update the output dir if it's still the default or doesn't contain language info
+                if "output" in str(output_dir) and language_name not in str(output_dir):
+                    # Keep the same parent, but add language info
+                    parent_dir = output_dir.parent
+                    output_dir = parent_dir / f"{language_name}_{language_info.get('code', '').replace('_', '-')}"
+                    print(f"Updated output directory to: {output_dir}")
+                    output_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
Do not add all of this code inside the `__main__.py` file. Also now it is named `dutch_nl-NL`. I'd rather have it called `piper-nl` instead.